src/main/java/cmu/pasta/mu2/diff/plugin/DiffGoal.java

```diff
@@ -188,6 +203,77 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             throw new MojoExecutionException("Invalid Mutation OptLevel!");
         }
 
+        // Set filters to default value of empty string
+        if (filters == null){
+            filters = "";
+        }
+
+        // Initialize empty list of filters
+        List<MutantFilter> filterList = new ArrayList<>();
+        int i = 0;
+        char[] filterChars = filters.toCharArray();
+        String current = "";
+        Filter f;
+        int len = filterChars.length;
+
+        // Parse input string (of format "filter1:arg1,arg2,...,filter2:arg1,arg2,...") to create list of filters
```

Comment:
How does the format distinguish between `filter2` and `arg2`? It looks look both are delimited by commas only. Do you want to add a semicolon in the format? E.g. `KLEASTEXEC:100;KRANDOM:5`? Or maybe continue to use commas but assume that there will be only one argument per filter?

Reply:
That's what I was initially planning to do but ';' seems to be an escape character. Since I'm currently just manually parsing for arguments, I figured it would be fine since for each filter we can just parse the appropriate number of arguments.